sssom/cli.py

```diff
@@ -89,6 +89,28 @@
     help="Fields.",
 )
 
+predicate_filter_option = click.option(
+    "-F",
+    "--mapping-predicate-filter",
+    multiple=True,
+    help="A list of predicates or a file path containing the list of predicates to be considered.",
+)
+
+
+def _get_list_of_predicates(predicate_filter: tuple) -> list:
+    
```

Comment:
What a out the case ["a:123", "list.txt", "b:234"]?

Reply:
Do we want this or enforce either a list OR individual CURIES?

Reply:
Changed the code to accept the above.